benchmarks/asv.conf.json

```diff
@@ -37,16 +37,16 @@
     ],
     "delegated_env_commands": {
         "c8a663a0": [
+            "ENV_PARENT={conf_dir}/.asv/env/nox313",
```

Comment:
This is awkward - we don't know what the new commit hash will be once this is merged. I didn't think of that 🙄

Reply:
Discussed with @ESadek-MO: if you just provide one of the commit hashes that is on your branch, and then we'll merge this one with a merge-commit (not a squash).

Reply:
Like this https://github.com/SciTools/iris/pull/6195/commits/bd6495c401f81e6a46a939ab1f3e0ef7c752df0b ?

Reply:
Almost, however there are now duplicate `3.13`, which is what is causing the benchmark failure.